pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -556,4 +556,41 @@ private static Comparator<Bytes> valueSorter(@NonNull final Comparator<Byte> byt
             return ((len > 0) ? 1 : -1);
         };
     }
+
+    /**
+     * Appends a {@link Bytes} object to this {@link Bytes} object, producing a new immutable {link Bytes} object.
+     * @param bytes2 The second {@link Bytes} object to append.
+     * @return A new {link Bytes} object containing the concatenated bytes and b.
+     * @throws BufferUnderflowException if the buffer is empty
+     * @throws IndexOutOfBoundsException If the given {@code offset} is negative or not less than Bytes.length()
+     */
+    @NonNull
+    public Bytes appendBytes(@NonNull final Bytes bytes) {
```

Comment:
can the name of both methods just be `append()` with just parameter types different

Reply:
Done.